test/unit/lib/cli/triage/index.test.js

```diff
@@ -95,5 +95,11 @@ describe('test/unit/lib/cli/triage/index.test.js', () => {
         register('@serverless/components', 'us');
       }
     }
+
+    it('Should recognize "@serverless/compose" if "serverless-compose.yml" file present', async () => {
```

Comment:
I think it's enough to add CLI name to L60, and it then should be automatically tested (no need for other `it`)